panther_docking/launch/docking.launch.py

```diff
@@ -50,6 +50,13 @@ def generate_launch_description():
         description=("Path to docking server configuration file."),
     )
 
+    log_level = LaunchConfiguration("log_level")
+    declare_log_level = DeclareLaunchArgument(
+        "log_level",
+        default_value="info",
+        description="Logging level",
+    )
```

Comment:
Could you create task to add such argument in all our launch files.

Reply:
1. Added
2. (y)